core/dbt/task/compile.py

```diff
@@ -40,6 +40,18 @@ def compile(self, manifest):
 
 
 class CompileTask(GraphRunnableTask):
+    # We add a new inline node to the manifest during initialization
+    # it should be removed before the task is complete
+    _inline_node_id = None
+
+    # TODO remove when stu's PR about skip adapter cache is merged
```

Comment:
As stated, this should be removed once Stu's work is done.

Reply:
Tracked in https://github.com/dbt-labs/dbt-core/issues/6526